components/panic_hook/src/lib.rs

```diff
@@ -60,8 +17,14 @@ pub fn recover_safe<F, R>(f: F) -> std::thread::Result<R>
 where
     F: FnOnce() -> R,
 {
-    mute();
+    // Mute the panic message
+    let original_hook = panic::take_hook();
```

Comment:
In the past, it will only mute the panic from current thread, but now it muts for all.